pkg/controller/controller.go

```diff
@@ -834,3 +838,21 @@ func bytesToGiQuantity(bytes int64) resource.Quantity {
 	stringQuantity := fmt.Sprintf("%v%s", num, suffix)
 	return resource.MustParse(stringQuantity)
 }
+
+func createVolumeRequestParameters(options controller.VolumeOptions, mappings map[string]string) map[string]string {
```

Comment:
maybe a UT for this?

Reply:
Thank you for your comment.

I will add unit tests for this, once we reach to the agreement that this specification is good to everyone.